src/metadata.cc

```diff
@@ -81,6 +81,22 @@ class MetadataWorker : public Nan::AsyncWorker {
         memcpy(baton->icc, icc, iccLength);
         baton->iccLength = iccLength;
       }
+      // IPTC
+      if (image.get_typeof(VIPS_META_IPCT_NAME) == VIPS_TYPE_BLOB) {
+        size_t iptcLength;
+        void const *iptc = image.get_blob(VIPS_META_ICC_NAME, &iptcLength);
+        baton->iptc = static_cast<char *>(g_malloc(iptcLength));
+        memcpy(baton->iptc, iptc, iptcLength);
+        baton->iptcLength = iptcLength;
+      }
+      // XMP
+      if (image.get_typeof(VIPS_META_XMP_NAME) == VIPS_TYPE_BLOB) {
+        size_t xmpLength;
+        void const *xmp = image.get_blob(VIPS_META_ICC_NAME, &xmpLength);
```

Comment:
```diff
-        void const *xmp = image.get_blob(VIPS_META_ICC_NAME, &xmpLength);
+        void const *xmp = image.get_blob(VIPS_META_XMP_NAME, &xmpLength);
```

Reply:
Uh, silly me.

Reply:
@lovell fixed. Tests passing locally. Waiting for CI